lessons/11-css-specificity/index.html

```diff
@@ -22,6 +24,16 @@
       nav a {
         color: green;
       }
+      nav a{
+        color:blue;
+        text-align: center;
+      }
+      header{
+        text-align: center;
+      }
+      p{
+        font-size: 14px;
+      }
```

Comment:
Clever! Because you defined the rules after the ones above, even though they are the same selectors they will take precedence over the earlier ones.

What would be another way to do it?